pgns/130311.js

```diff
@@ -19,6 +19,9 @@ module.exports = [
         '.humidity'
       )
     },
+	filter: function (n2k) {
+      return n2k.fields['Humidity']
```

Comment:
This will be falsy if the value is 0. Not a huge deal, as it probably never is, but a typeof != 'undefined' test would be better.